init/eessi_defaults

```diff
@@ -8,5 +8,8 @@
 # license: GPLv2
 #
 
-export EESSI_CVMFS_REPO="${EESSI_CVMFS_REPO_OVERRIDE:=/cvmfs/pilot.eessi-hpc.org}"
-export EESSI_PILOT_VERSION="${EESSI_PILOT_VERSION_OVERRIDE:=2023.06}"
+export EESSI_CVMFS_REPO="${EESSI_CVMFS_REPO_OVERRIDE:=/cvmfs/software.eessi.io}"
+export EESSI_VERSION="${EESSI_VERSION_OVERRIDE:=2023.06}"
+# use archdetect by default, unless otherwise specified
+export EESSI_USE_ARCHDETECT="${EESSI_USE_ARCHDETECT:=1}"
```

Comment:
I just want to point out here that this flips the default from using `archspec` (like we do in `pilot.eessi-hpc.org`) to our pure bash `archdetect` alternative.

Not only is that significantly faster (no overhead of starting a Python interpreter), but it will also help to bypass some shortcomings in `archspec` (see #320 + #322).

Reply:
:+1: